stable/artifactory/values.yaml

```diff
@@ -1299,3 +1299,16 @@ filebeat:
     output:
       logstash:
          hosts: ["{{ .Values.filebeat.logstashUrl }}"]
+
+# FLUENT BIT SIDECAR CONFIGURATION
+fluentbit:
+  enabled: false
+  name: artifactory-fluent-bit
+  image:
+    repository: fluent/fluent-bit
+    version: 1.5.5
+  resources: {}
+  # TO ENABLE READINESS / LIVENESS ADD HTTP INPUT
+  # IN FLUENT BIT CONF AND CURL ENDPOINT
+  #readinessProbe:
```

Comment:
Please add a full example of how to configure the probes. I think this should be enabled by default.
Also, lets try to keep the convention of probes' configuration, as we did for other products. e.g.:
```
readinessProbe:
  enabled: true
  configuration: |
    ...
```